Form-Controls/styles.css

```diff
@@ -0,0 +1,127 @@
+:root {
+    --white:#FFF;
+    --black:#000;
+    --db:#E6DACD;
+    --lb:#F4ECE6;
+    --rb:#0150FD;
+    --header-font: 'Poppins', sans-serif;
+    --gen-font: 'Quicksand', sans-serif;
+}
+
+body {
+    margin: 0;
+}
+
+header {
+    background: var(--white);
+    height: 100px;
+}
+
+header div {
+    text-align: center;
+    padding-top: 1rem;
+}
+
+h1 {
+    font-family: var(--header-font);
+    margin: 0;
+}
+
+section {
+    background: var(--db);
+    height: 1000px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.section__container {
+    display: flex;
+    justify-content: center;
+    height: 800px;
+    width: 60%;
```

Comment:
60% of what? What if the screen was 320px wide. Would you only want 192px for your form?

In general, instead of setting explicit widths and heights, set maximums and minimums and let your content flow/pour itself into many viewports. This strategy will make all your designs easier to manage. 

Think of your layout as a bento box and each component as a delicious dish you are pouring into the available compartments. In web design, we never truly know the shape and size of the bento box, so it's better to learn how to describe the relationships between objects than the objects directly.

Look into `clamp` and `calc` to help you do this.

Reply:
Perfect, I learnt max(), min(), clamp() and calc(). That is what I lacked so far. Great thank you